autoload/sy.vim

```diff
@@ -24,9 +31,10 @@ function! sy#start() abort
     return
   endif
 
-
   function! s:chdir()
```

Comment:
Maybe we should just move this to `sy#util#chdir`.  I don't see a reason to cache this in the `sy_info` dict.